python/ml4ir/base/config/default_evaluation_config.yaml

```diff
@@ -0,0 +1,22 @@
+power_analysis:
+  # a list of input feature names (comma separated) used to aggregate results. Leave blank if not used
+  group_by:
```

Comment:
@mohazahran Isn't this already captured in `FeatureConfig`? I don't think you need this here.

Reply:
ping

Reply:
yes, but we there is a usecase where we can run multiple evaluations and want to change the group keys

Reply:
Then do that in the feature config. Why would we need to do it here?

Reply:
I'm only pushing you because it leads to duplication and unnecessary tech debt.